### ISSUE TITLE
Some keys are not properly recognized

### ISSUE THREAD

**Issue by Seblor** [OhMyGuus/better-crewlink-mobile#10]
On CrewLink, I am used to have the <kbd>Right Alt</kbd> and <kbd>Right Ctrl</kbd> key to respectively mute and deafen myself. However, Better-CrewLink does not make the distinction between <kbd>Left Ctrl</kbd> and <kbd>Right Ctrl</kbd>, and is mistaking <kbd>Right Alt</kbd> for <kbd>Ctrl</kbd>.

**Comment by OhMyGuus** [comment]
This issue is on the wrong github, for issues on the PC version u can create a issue at: 
https://github.com/OhMyGuus/BetterCrewLink/issues

--- about the issue tho --- 
I made a decision about changing the keyboard shortcut handler and moved it from IOHOOK ( the one that crewlink is using) to something I made using getkeystateAsync(). This because iohook wasn't working correctly (it didn't register when u had crewlink selected & it added a slight lag to my mouse/keyboard for a few seconds. 

The down side of this change is that the determination of rightcontrol and left control stopped working, I stopped looking at it because I couldn't figure it out. I might however check it again soon.
